test/integration/helpers.bash

```diff
@@ -230,8 +230,8 @@ function start_docker() {
 			${DOCKER_IMAGE}:${DOCKER_VERSION} \
```

Comment:
If we're not directly interested in testing against _master_, we could also  consider using the `docker:dind` images from Docker Hub; https://hub.docker.com/_/docker